Rmarkdown/Multiple_Ascending_Dose_PD_ordinal.Rmd

```diff
@@ -112,13 +113,41 @@ gg <- gg + facet_grid(.~TRTACT_low2high)
 gg <- gg + guides(fill = guide_legend(""))
 gg <- gg + xgx_annotate_status(status)
 gg
+
+gg2 <- ggplot(data = pd_data, aes(x = PROFDAY, response = ORDINAL_LEVELS_high2low, color = ORDINAL_LEVELS_high2low)) 
+gg2 <- gg2 + xgx_stat_ci(distribution = "ordinal", 
+                         geom = list("line", "point","errorbar"), 
+                         position = position_dodge(width = 12), alpha = 0.5) 
+gg2 <- gg2 + scale_y_continuous(labels = scales::percent)
+gg2 <- gg2 + labs(x = time_label, y = pd_response_label)
+gg2 <- gg2 + facet_grid(.~TRTACT_low2high)
+gg2 <- gg2 + scale_fill_brewer(palette = 6)
+gg2 <- gg2 + guides(color = guide_legend(""),fill = guide_legend(""))
+gg2 <- gg2 + xgx_annotate_status(status)
+gg2
 ```
 
+### Percent of subjects by response category over time, colored by dose, faceted by response category
```

Comment:
As in the PKPD, these curves disappear when the % of patients in that category goes to zero.  I think that's not the right behavior and you want the plot to show these zeros.